playbooks/tasks/cloudhost.yml

```diff
@@ -31,22 +31,20 @@
 
 - name: Ensure APF Start-on-boot
   service:
-    name="apf"
-    enabled="yes"
-    state="started"
+    name: 'apf'
```

Comment:
we do name="apf" style nearly everywhere else across our ansible stuff.

i vote stick with that style unless we wanna explicitly change over, and in that case, we change over.

also style changes and functional changes should be in separate commits ideally

Reply:
changed to just be the reboot section